packages/js/e2e-environment/bin/scaffold.js

```diff
@@ -24,12 +24,24 @@ const {
 } = require( '../utils/scaffold' );
 
 const args = process.argv.slice( 2 );
-const [ command, packageName ] = args;
+const command = args[0];
+let packageName = null;
+
+if (args[1].substr(0, 2) !== '--') {
```

Comment:
This check here causes a break when I run the ` pnpx wc-e2e install ` command. Doing a check to ensure that `args[1]` isn't undefined seems to do the trick. 
```suggestion
if ( args[1] && args[1].substr(0, 2) !== '--') {
```

Reply:
Thanks for the suggestion, @MrJnrman! I can see how this is necessary, thank you. If there's no package name, I'm potentially calling `substr` on `undefined`.

I'm just curious to understand why I didn't face this error in my local test when running `pnpx wc-e2e install` :thinking: I'm running Node.js version v16.13.1 for what it's worth